data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -305,26 +310,75 @@
             speaker=narrator
             image=$unit.profile
             caption= _ "Elvish Shaman"
-            message= _ "The Shaman is a fairly weak unit, but she has the ability to <i>heal</i> friendly units around her. She also has a special attack which <i>slows</i> enemies, halving the damage they do for one turn."
+            message= _ "The Shaman has the ability to <i>heal</i> friendly units around her. She also has a special attack which <i>slows</i> enemies, halving the damage they do for one turn. However, physically she’s weaker than the fighters and archers, so be careful when sending her into battle."
         [/message]
 
         [message]
             speaker=narrator
-            caption= _ "Abilities and Weapon Specials"
+            caption= _ "The <i>Heal</i> Ability"
             image=wesnoth-icon.png
-            message= _ "Certain unit types have special <i>abilities</i>. Unlike traits, which usually provide small buffs or debuffs, abilities give units special behaviors that normal units do not have. In this case, the Elvish Shaman has the Heal ability, which allows her to heal allied adjacent units 4 hitpoints at the start of each turn. Unit types with abilities, especially ones like Heal, can be very useful, so be careful when sending them into battle."
+            message= _ "At the start of each turn, allied units adjacent to a Shaman will recover 4 hitpoints. This is only half of the amount that a village would provide, but you rarely have a village near your front-line. The Shaman doesn’t gain experience from healing, only from combat, but advancing her is worthwhile — her second-level Elvish Druid advancement has healing equivalent to a village.
```

Comment:
```suggestion
            message= _ "At the start of each turn, allied units adjacent to a Shaman will recover 4 hitpoints, which is half the amount that a village provides. The Shaman doesn’t gain experience from healing, but advancing her is worthwhile — her second-level Druid advancement has healing equivalent to a village.
```

Reply:
I'll take the "which" change, but disagree with removing the note about xp from combat - the extra words make it easier to understand.